app/controllers/feedbacks_controller.rb

```diff
@@ -0,0 +1,28 @@
+class FeedbacksController < ApplicationController
+  before_action -> { authorize! @feedback, with: FeedbackPolicy }
+
+  def new
+    @feedback = Feedback.new
+  end
+
+  def create
+    result = Feedbacks::Action.new.call(feedback_params)
+
+    if result.success?
+      flash.now[:notice] = "Feedback sent successfully"
+      render :thanks
+    else
+      flash.now[:alert] = result.failure[:message]
+      render :new
+    end
+  end
+
+  def thanks
```

Comment:
Why you need it?

Reply:
As it seemed to me what is required in the controller to render the view, but, of course, no. It redundant